### ISSUE TITLE
Add missing og tags.

### ISSUE THREAD

**Issue by tiswas247** [bionic-gpt/bionic-gpt#953]
The four required Open Graph tags for every page are og:title, og:type, og:image, and og:url.

We need an image for og:image

Examples

- https://cdn.prod.website-files.com/67d1b10ea1804fdfad7d7a38/681cbd5b02e23f07e6225c6f_open-graph-home.webp

What shall we set og:url to?

## Missing alt text 

The github shield
